frontend/js/src/playlists/Playlist.tsx

```diff
@@ -108,7 +108,10 @@ export default class PlaylistPage extends React.Component<
   };
 
   createWebsocketsConnection = (): void => {
-    this.socket = io(`${window.location.origin}`, { path: "/socket.io/" });
+    const { websocketsUrl } = this.context;
+    this.socket = io(`${websocketsUrl || window.location.origin}`, {
```

Comment:
I suppose at some point we had a reason to use a template literal, but now we can simply:
```suggestion
    this.socket = io(websocketsUrl || window.location.origin, {
```